logger/logger.go

```diff
@@ -82,7 +82,7 @@ func NewSimpleLoggerFromMinimalLogger(minLog MinimalLogger, level string) Logger
 // level. The level can be, in order: "error", "info", "debug". It takes an
 // io.Writer.
 func NewSimpleLogger(w io.Writer, level string) Logger {
-	flags := log.Ldate | log.Ltime | log.Lmicroseconds
+	flags := log.Ldate | log.Ltime 
```

Comment:
Why this change?

Reply:
Because the push server logs are very wide, as the system journal adds another timestamp in front. And for a push service we do not need that precision :) - Just needs to be checked on the client side, if it is the same or we loose fidelity.